edm4eic.yaml

```diff
@@ -372,39 +391,40 @@ datatypes:
-      - float             theta             // Track polar angle [rad]
-      - float             phi               // Track azimuthal angle [rad]
-      - float             qOverP            // [e/GeV]
-      - edm4eic::Cov3f    momentumError     // Covariance on theta, phi and qOverP
-      - float             time              // Track time [ns]    
-      - float             timeError         // Error on the time
-      - float             charge            // Particle charge
+      - int32_t              type              // Type of track parameters (-1/seed, 0/head, ...)
+      - uint64_t             surface           // Surface for bound parameters (geometryID)
+      - edm4hep::Vector2f    loc               // 2D location on surface
+      - float                theta             // Track polar angle [rad]
+      - float                phi               // Track azimuthal angle [rad]
+      - float                qOverP            // [e/GeV]
+      - float                time              // Track time [ns] 
+      - edm4eic::Cov6f       covariance        // Full covariance in basis [l0,l1,theta,phi,q/p,t]
+  
     OneToOneRelations:
       - edm4eic::Trajectory trajectory      // Trajectory associated with these track parameters
 
+
   edm4eic::Track:
     Description: "Track information at the vertex"
-    Author: "S. Joosten"
+    Author: "S. Joosten, J. Osborn"
     Members:
-      - int32_t           type              // Flag that defines the type of track
-      - float             chi2              // Total chi2 (sum) of the track fit
-      - int32_t           ndf               // Numbers of degrees of freedom of the track fit
-      - edm4hep::Vector3f momentum          // Track 3-momentum at the vertex [GeV]
-      - edm4eic::Cov3f    momentumError     // Covariance matrix on the momentum
-      - float             time              // Track time at the vertex [ns]
-      - float             timeError         // Error on the track vertex time
-      - float             charge            // Particle charge
+      - int32_t            type                           // Flag that defines the type of track
+      - edm4hep::Vector3f  position                       // Track 3-position at the vertex 
+      - edm4hep::Vector3f  momentum                       // Track 3-momentum at the vertex [GeV]
+      - edm4eic::Cov6f     positionMomentumCovariance     // Covariance matrix in basis [x,y,z,px,py,pz]
+      - float              time                           // Track time at the vertex [ns]
+      - float              timeError                      // Error on the track vertex time
+      - float              charge                         // Particle charge
+      - float              chi2                           // Total chi2
+      - uint32_t           ndf                            // Number of degrees of freedom
+      - uint32_t           pid                            // PDG particle ID hypothesis
```

Comment:
Other data types use `pdg` if I recall correctly.

Reply:
I personally prefer pid, but also like consistency :)

Reply:
PDG code should be `int32_t` (signed integer)